### PR TITLE
4.20 Support

### PR DESCRIPTION
**Pull request by Rbn3D**
Hi,

This PR adds support to UE 4.20 (tested on UE 4.20.3, Win64). 
It includes [this guide](https://github.com/Rbn3D/OceanProject/blob/4.20/INSTALL.md) explaining how to install it into your project and package it sucessfully.

### CODE REVIEW

**Comment by EvoPulseGaming** [comment]
If I understood your 8112e85 commit.... : This will never be accepted as long as its an engine module, this will only work for all uses in the most common case is if its a project plugin. 

Or does this just make it compatible as an engine module?

File: OceanProject.uproject [hunk 1]
```diff
@@ -9,5 +9,11 @@
 			"Type": "Runtime",
 			"LoadingPhase": "Default"
 		}
+	],
```
Comment (EvoPulseGaming) [inline OceanProject.uproject:12]: Why are we referencing our self in this plugin? Is this for the engine module compatibility? Does this still work as a standalone plugin?

File: INSTALL.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+* Follow [this guide from the wiki](https://github.com/UE4-OceanProject/OceanProject/wiki/How-to-add-to-your-project) to install into your own project.
+
+
+#### How to package (and include OceanPlugin in the build)
+
+* Close your project and UE4
+
+* Move the `OceanPlugin` folder (which is on `Plugins` folder) to `Program Files/Epic Games/UE_4.20/Engine/Plugins/Runtime`
+
+* Edit your `.uproject` and add this:
```
Comment (EvoPulseGaming) [inline INSTALL.md:22]: I'm confused here, should we have a single .uproject file that can work in the editor and for packaging? A default standard would be approved here. See comment in the .uproject file

File: Plugins/OceanPlugin/Source/OceanPlugin/OceanPlugin.Build.cs [hunk 3]
```diff
@@ -35,7 +35,7 @@ private string ThirdPartyPath
 
     public OceanPlugin(ReadOnlyTargetRules Target) : base(Target)
     {
-        PrivateIncludePaths.AddRange(new string[] { "OceanPlugin/Private" });
+        PrivateIncludePaths.AddRange(new string[] { Path.Combine(ModuleDirectory, "Private") });
```
Comment (EvoPulseGaming) [inline Plugins/OceanPlugin/Source/OceanPlugin/OceanPlugin.Build.cs:38]: Is this still compatible as a standalone plugin? Without making this an engine module?

Comment (Rbn3D) [reply Plugins/OceanPlugin/Source/OceanPlugin/OceanPlugin.Build.cs:38]: I think this is the way UE wants the includes paths starting from 4.20, and yes, will work as project plugin as well as engine plugin.

Comment (EvoPulseGaming) [reply Plugins/OceanPlugin/Source/OceanPlugin/OceanPlugin.Build.cs:38]: Alright cool, thank you

File: OceanProject.uproject (hunk 1 above)
Comment (Rbn3D) [reply OceanProject.uproject:12]: Mmm, although I'm sure it works this way, not sure if it's completely need... But I think it's just a way to tell the project that has a dependence with the plugin.

File: INSTALL.md (hunk 2 above)
Comment (Rbn3D) [reply INSTALL.md:22]: I think yes, it's working in both ways.

**Comment by Rbn3D** [comment]
Thank you for accepting this.

File: INSTALL.md (hunk 2 above)
Comment (EvoPulseGaming) [reply INSTALL.md:22]: Perfect

File: OceanProject.uproject (hunk 1 above)
Comment (EvoPulseGaming) [reply OceanProject.uproject:12]: Yes of course, I'm not sure how the project got away with this before xD Thanks for catching that

**Comment by EvoPulseGaming** [comment]
Thank you for taking your time to keep this project alive

**Pull request merged**